flake.nix

```diff
@@ -8,7 +8,7 @@
 
   inputs.nixpkgs.url = "nixpkgs/nixos-unstable";
   inputs.flake-utils.url = "github:numtide/flake-utils";
-  inputs.nickel.url = "github:tweag/nickel/stable";
+  inputs.nickel.url = "github:tweag/nickel";
```

Comment:
I think that's left over from testing. It would probably make sense to merge this PR into an `unstable` branch on `nickel-lang.org` and change this input over before integrating into `master`.